skyportal/handlers/api/obj.py

```diff
@@ -0,0 +1,51 @@
+from baselayer.app.access import auth_or_token, AccessError
+from baselayer.app.env import load_env
+from ..base import BaseHandler
+from ...models import DBSession, Obj
+
+_, cfg = load_env()
+
+
+class ObjHandler(BaseHandler):
+    @auth_or_token  # ACLs will be checked below based on configs
+    def delete(self, obj_id):
+        """
+        ---
+        description: Delete an Obj
+        tags:
+          - objs
+        parameters:
+          - in: path
+            name: obj_id
+            required: true
+            schema:
+              type: string
+        responses:
+          200:
+            content:
+              application/json:
+                schema: Success
+          400:
+            content:
+              application/json:
+                schema: Error
+        """
+        try:
+            obj = Obj.get_if_accessible_by(
+                obj_id,
+                self.current_user,
+                mode='delete',
+                raise_if_none=False,
```

Comment:
```suggestion
                raise_if_none=True,
```
otherwise the `DBSession().delete(obj)` below could error if an invalid ID is specified

Reply:
Good catch, I turned this off for testing purposes and forgot to turn it back on